.travis.yml

```diff
@@ -46,6 +52,24 @@ jobs:
           -DCMAKE_BUILD_TYPE=Debug -DGLOW_WITH_OPENCL=OFF -DGLOW_WITH_CPU=ON
           -DCMAKE_PREFIX_PATH=/usr/lib/llvm-6.0/include/
           ../
+      script:
+        - function keep_alive() { while true; do echo -en "\a"; sleep 540; done }
```

Comment:
Can we push all of this stuff into a script?  We should simplify the travis file.